locallibs/productbuild.py

```diff
@@ -0,0 +1,31 @@
+import os
+import shutil
+import subprocess
+import sys
+
+class ProductBuildException(Exception):
+    '''Error when creating pkg'''
+    pass
+
+def generate(info, createuserpkg_dir):
+    # Hide and rename created component package
+    component_pkg = os.path.dirname(info[u'destination_path']) + "/.tmp.pkg"
+    shutil.move(os.path.expanduser(info[u'destination_path']), component_pkg)
```

Comment:
If something went wrong with an earlier run, .tmp.pkg might still be around. You might want to account for this possibility.

Reply:
Well, when the package is created, those actions are wrapped in a try block so that would clean up the .tmp file if something were to go wrong. As far as I see, the only other thing that could go wrong is renaming the package, but if that would fail, there wouldn't be a .tmp file to clean up so I'm not sure what you mean here, am I missing something?

Reply:
Yes, you're right, the `finally` clause makes sure it's cleaned up.